libpod/info.go

```diff
@@ -178,3 +186,26 @@ func (r *Runtime) GetOCIRuntimeVersion() (string, error) {
 	}
 	return strings.TrimSuffix(output, "\n"), nil
 }
+
+// GetHostDistributionInfo returns a map containing the host's distribution and version
+func (r *Runtime) GetHostDistributionInfo() map[string]string {
+	dist := make(map[string]string)
+
+	f, err := os.Open("/etc/os-release")
+	defer f.Close()
```

Comment:
Throw this after the error return so we don't try and close if Open returned an error

Reply:
fixed ...